libsolidity/interface/StandardCompiler.cpp

```diff
@@ -91,6 +92,21 @@ Json::Value formatErrorWithException(
 	return formatError(_warning, _type, _component, message, formattedMessage, location);
 }
 
+/// @_hash contains a Keccak256 hash with the 0x prefix
```

Comment:
No, not as far as I know. `@a` tells the doxygen parser that the following word is a function argument.